common/defaults.go

```diff
@@ -307,30 +339,31 @@ vs-ssh.visualstudio.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC7Hr1oTWqNqOlzGJOf
 )
 
 // DefaultLabels returns the default set of labels for controllers.
-func DefaultLabels(name, component string) map[string]string {
+func DefaultLabels(resourceName, instanceName, component string) map[string]string {
```

Comment:
Can we put one of `resourceName` and `instanceName` as optional arg? I see a lot of `Labels:    common.DefaultLabels(cr.Name, cr.Name, ""),` which is quite confusing.

Reply:
we should pass in both `resourceName` and `instanceName` because they go into different labels, serving different purposes

> Labels:    common.DefaultLabels(cr.Name, cr.Name, ""),
Yeah, we are going to change this behavior as a part of the rewrite 
resource name and instance name shouldn't be the same